tritium-registry/src/test/java/com/palantir/tritium/metrics/registry/TaggedMetricRegistryTest.java

```diff
@@ -96,18 +74,21 @@ public void before() {
         Mockito.verify(mockSupplier, times(2)).get();
     }
 
-    @Test
-    public void testCounter() {
+    @ParameterizedTest
+    @MethodSource("com.palantir.tritium.registry.test.TestTaggedMetricRegistries#registries")
```

Comment:
Thoughts on extracting `"com.palantir.tritium.registry.test.TestTaggedMetricRegistries#registries"` to a constant in `TestTaggedMetricRegistries`?

Reply:
Yeah, can extract these to constants. I can probably tweak the projects’ test dependencies to use a single implementation as well.